assets/js/script.js

```diff
@@ -29,4 +31,295 @@ closeAbout.addEventListener('click', () => {
+    buttonLive: 'https://wdavidcch.github.io/portfolio/',
+    buttonSource: 'https://github.com/wdavidcch/portfolio',
+  },
+  {
+    image: './assets/img/face-project.jpg',
+    title: 'Facebook 360',
+    languages: ['Ruby on Rails', 'CSS', 'JavaScript', 'Html'],
+    description:
+      'Exploring the future of media in Facebook`s first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    buttonLive: 'https://wdavidcch.github.io/portfolio/',
+    buttonSource: 'https://github.com/wdavidcch/portfolio',
+  },
+  {
+    image: './assets/img/uber-project.jpg',
+    title: 'Uber Navigation',
+    languages: ['Ruby on Rails', 'CSS', 'JavaScript', 'Html'],
+    description:
+      'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    buttonLive: 'https://wdavidcch.github.io/portfolio/',
+    buttonSource: 'https://github.com/wdavidcch/portfolio',
+  },
+  {
+    image: './assets/img/laptop-project.jpg',
+    title: 'Profesional Art Printing Data More',
+    languages: ['Ruby on Rails', 'CSS', 'JavaScript', 'Html'],
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard',
+    buttonLive: 'https://wdavidcch.github.io/portfolio/',
+    buttonSource: 'https://github.com/wdavidcch/portfolio',
+  },
+];
+
+// projects dynamically appears on the page
+
+const cardsContainer = document.querySelector('.cards-container');
+cardsContainer.innerHTML = `<article class="product-card">
+<img src="./assets/img/projects.jpg" alt="img" />
+<div class="product-info">
+    <h2>Keeping track of hundreds of components</h2>
+    <div>
+        <ul class="leng">
+            <li>Ruby on Rails</li>
+            <li>Css</li>
+            <li>JavaScript</li>
+            <li>Html</li>
+        </ul>
+    </div>
+    <button type="button" class="project-button" value="See">See Project</button>
+</div>
+</article>
+
+<article class="product-card">
+<img src="./assets/img/medicalproject.jpg" alt="img" />
+<div class="product-info">
+    <h2>Medical Illustration Sets</h2>
+    <div>
+        <ul class="leng">
+            <li>Ruby on Rails</li>
+            <li>Css</li>
+            <li>JavaScript</li>
+            <li>Html</li>
+        </ul>
+    </div>
+    <button type="button" class="project-button2" value="See">See Project</button>
+</div>
+</article>
+
+<article class="product-card">
+<img src="./assets/img/girlproject.jpg" alt="img" />
+<div class="product-info">
+    <h2>Multi Posts Stories</h2>
+    <div>
+        <ul class="leng">
+            <li>Ruby on Rails</li>
+            <li>Css</li>
+            <li>JavaScript</li>
+            <li>Html</li>
+        </ul>
+    </div>
```

Comment:
Also, kindly edit the spacing of the popup in the `mobile version` to stick to the design provided.

| Your Design | Figma Design |
|-------------|---------------|
| <img width="406" alt="image" src="https://user-images.githubusercontent.com/34578631/174144191-b5d0c156-0eb0-4e42-98cc-0acc1dede722.png"> | <img width="365" alt="image" src="https://user-images.githubusercontent.com/34578631/174144274-c612bb51-8122-4f39-a9a7-70ecce970d06.png"> |

**Hint:** You may have to remove the wrapping `div` styling of `justify-content: space-between;` and giving the spacing in another way might help.

Reply:
Thank you I will use grid to fix that.